QStreak/View Controllers/Account Setup/AccountSetupViewController.swift

```diff
@@ -14,11 +14,9 @@ class AccountSetupViewController: UIViewController {
 
     @IBOutlet private weak var zipCodeTextField: UITextField!
 
-    @IBOutlet private weak var ageTextField: UITextField!
+    @IBOutlet private weak var continueButton: QButton!
```

Comment:
Can you subclass `QButton` and do the styling there? That way if designs change we can make any changes in one location.